control/autoware_pid_longitudinal_controller/src/pid_longitudinal_controller.cpp

```diff
@@ -1107,11 +1107,13 @@ double PidLongitudinalController::applyVelocityFeedback(const ControlData & cont
     (vehicle_is_moving || (m_enable_integration_at_low_speed && vehicle_is_stuck)) &&
     is_under_control;
 
+  const bool erase_integral = !enable_integration;
```

Comment:
Any reason to use the negative enable_integration here?
Using enable_integration consistently is easier to understand the code.

Reply:
fixed in 0555ea54cec056d80840515a5a92ee328643a93b